packages/edit-site/src/components/global-styles/font-library-modal/index.js

```diff
@@ -44,11 +46,23 @@ function FontLibraryModal( {
 	defaultTabId = 'installed-fonts',
 } ) {
 	const { collections, setNotice } = useContext( FontLibraryContext );
+	const canUserCreate = useSelect( ( select ) => {
+		const { canUser } = select( coreStore );
+		return canUser( 'create', 'font-families' );
+	}, [] );
+	const fontUploadsEnabled = useSelect( ( select ) => {
+		const { getEditorSettings } = select( editorStore );
+		return getEditorSettings().fontUploadsEnabled;
+	}, [] );
+
+	const tabs = [ DEFAULT_TAB ];
 
-	const tabs = [
-		...DEFAULT_TABS,
-		...tabsFromCollections( collections || [] ),
-	];
+	if ( canUserCreate && fontUploadsEnabled ) {
```

Comment:
I think this should check only `canUserCreate`. Otherwise, we will disallow installing system fonts.
```suggestion
	if ( canUserCreate ) {
```

Reply:
@matiasbenedetto Are you able to expand on this a little. I'm not seeing the ability to add System fonts when running Gutenberg. 

With the recent changes I can see the Google Fonts tabs but without the ability to upload fonts I can go through the process to install only to hit an inactive install button once I try to make use of them. It's a confusing experience.

![Screen Shot 2024-02-26 at 9 56 56 am](https://github.com/WordPress/gutenberg/assets/519727/e62535d2-a98d-49bc-ae69-a16706f1b45c)